test/handlers/FranchiserFactoryHandler.sol

```diff
@@ -25,9 +25,13 @@ contract FranchiserFactoryHandler is Test {
     // Handler ghost mapping to track the number of calls to each handler function
     mapping(bytes32 => CallCounts) public calls;
 
-    // Handler-updated booleans to track balance and voting power updates were correctly calculated
-    bool public balances_updated_correctly;
-    bool public voting_powers_updated_correctly;
+    // Handler ghost address set and mapping to track the balance of every address that is used in the invariant tests
+    EnumerableSet.AddressSet private accountBalanceAddresses;
+    mapping(address => uint256) private accountBalances;
```

Comment:
i'm on the fence about whether storing account balances / vote balances here makes sense. do we already get this state via the franchiser contract itself? check this section of this article out and let me know whether it could apply here: https://mirror.xyz/horsefacts.eth/Jex2YVaO65dda6zEyfM_-DXlXhOWCAoSpOx5PLocYgw#:~:text=So%20how%20can,all%20the%20balances.

defer to you guys